src/platform/site-wide/user-nav/components/HelpMenu.jsx

```diff
@@ -3,23 +3,37 @@ import React from 'react';
 
 import DropDownPanel from '@department-of-veterans-affairs/formation/DropDownPanel';
 import IconHelp from '@department-of-veterans-affairs/formation/IconHelp';
+import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-flag';
 
 class HelpMenu extends React.Component {
   render() {
+    const buttonText = isBrandConsolidationEnabled ? 'Contact Us' : 'Help';
```

Comment:
Ah maybe this is misleading, but `isBrandConsolidationEnabled` is actually a function. If you feel that it would make more sense as a boolean though I'm onboard with that change.